tests/forte/processors/data_augment/algorithms/eda_processors_test.py

```diff
@@ -194,6 +206,12 @@ def test_random_delete(self, texts):
                 'kwargs': {}
             },
             "alpha": 0.5,
+            'augment_pack_names': {
+                'type': '',
+                'kwargs': {
+                    'input_src': 'augmented_input_src'
+                }
+            }
         }
 
         delete_processor = RandomDeletionDataAugmentProcessor()
```

Comment:
So in these tests we create some multipacklinks, but it seems that they don't affect the tests. Do we need them here?

Reply:
No. I removed them. Thanks!